### PR TITLE
Add Nginx Proxy Manager to Nginx networking webpage

### PR DESCRIPTION
**Pull request by Ladrien-git**
Nginx Proxy Manager is quite restrictive on what it allows you to do, and because of that the regular Nginx guide does not really apply without spending hours changing settings to see what works.

So I made a guide specifically for Nginx Proxy Manager.

### CODE REVIEW

File: docs/general/networking/nginxproxymanager.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    
+    # Tell browsers to use per-origin process isolation
+    add_header Origin-Agent-Cluster "?1" always;
+
+    # Content Security Policy
+    # See: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
+    # Enforces https content and restricts JS/CSS to origin
+    # External Javascript (such as cast_sender.js for Chromecast) must be whitelisted.
+    # NOTE: The default CSP headers may cause issues with the webOS app
+    add_header Content-Security-Policy "default-src https: data: blob: http://image.tmdb.org; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' https://www.gstatic.com/cv/js/sender/v1/cast_sender.js https://www.gstatic.com/eureka/clank/95/cast_sender.js https://www.gstatic.com/eureka/clank/96/cast_sender.js https://www.gstatic.com/eureka/clank/97/cast_sender.js https://www.youtube.com blob:; worker-src 'self' blob:; connect-src 'self'; object-src 'none'; frame-ancestors 'self'";
```
Comment (JPVenson) [inline docs/general/networking/nginxproxymanager.md:52]: youtube?

**Comment by nielsvanvelzen** [comment]
I don't think we need a dedicated page for nginx proxy manager, it would fit better as a subsection in the existing nginx page.

File: docs/general/networking/nginx.md [hunk 2]
```diff
@@ -388,3 +391,102 @@ error_page 460 http://your-page-telling-your-limit/;
 ```
 
 [See here for more](https://www.nginx.com/blog/rate-limiting-nginx/)
+
+## Nginx Proxy Manager
+
+[Nginx Proxy Manager](https://nginxproxymanager.com/) (pronounced "engine X") was created by jc21 in 2016 and provides an easy-to-use web GUI for Nginx.
```
Comment (nielsvanvelzen) [inline docs/general/networking/nginx.md:397]: Pronounciation is wrong as it skips the "proxy manager" part. I also don't think it's worth mentioning that part. Just tell us what the tool is, a jellyfin documentation reader doesn't care who made it and how they pronounce it.

File: docs/general/networking/nginx.md [hunk 3]
```diff
@@ -388,3 +391,102 @@ error_page 460 http://your-page-telling-your-limit/;
...
+
+### Nginx Proxy Manager from a subdomain (jellyfin.example.org)
```
Comment (nielsvanvelzen) [inline docs/general/networking/nginx.md:399]: This title could be skipped

File: docs/general/networking/nginx.md [hunk 4]
```diff
@@ -388,3 +391,102 @@ error_page 460 http://your-page-telling-your-limit/;
...
+
+:::tip
+
+The default X-Frame-Options header may cause issues with the webOS app, causing it to remain stuck at a black screen. If enabled, the default Content Security Policy may also cause issues.
+
+:::
```
Comment (nielsvanvelzen) [inline docs/general/networking/nginx.md:405]: Why duplicate this tip when already mentioned above in this same document?

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Comment by nielsvanvelzen** [comment]
Thanks!

**Pull request merged**